syntax = "proto3";

message MsgDeliveryRequest {
  int32 id = 1;
  int32 event_id = 2;
  string interface = 3;
  int32 money = 4;
  int32 balance = 5;
}

message MsgDeliveryResponse {
  int32 id = 1;
  int32 event_id = 2;
  int32 balance = 3;
  string result = 4;
}

service Branch {
  rpc MsgDelivery(MsgDeliveryRequest) returns (MsgDeliveryResponse) {}
}
